validation.py

```diff
@@ -167,7 +167,7 @@ def __build_run_cmd(self):
                 cmd.extend(['-v', volume])
 
         # Keyfile
-        cmd.append('-v%s:%s:ro' % (self.__params['keyfile'],
+        cmd.append('-v%s:%s:z' % (self.__params['keyfile'],
```

Comment:
are we sure we want to allow write access there??

Reply:
so if we use the "root" user in the container, we should be able to enforce "read-only" on the key. To be tested.

Reply:
Done